CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - SSL support (#155)
 - IPROTO_PUSH messages support (#67)
 - Public API with request object types (#126)
+- Context support for request objects (#48)
```

Comment:
I'm not very good in basic Go concepts. What's the main point of this? What do we obtain from using the context? What we can use if for? And how we possibly get increase in performance? For me it seems that it is more expected to drop because of increasing complication.

Reply:
Basically, one of the essential points is not to let your system to collect lots of requests on some heavy operations like going to another API or querying some DB. There are some other patterns of using this, but in general, the context mechanism is more widely than using only timeouts. Another question is what patterns may be useful for developing with tarantool.

What about performance, the current implementation doesn't affect it at all, and the complication of code also seems not to be increased a lot.

Reply:
The main idea of the patch is to provide a mechanism, some kind of API, for removing the Future, that is associated with a request, from the internal queue. There is already a way to do it - with a single timeout for every Future object, but sometimes it may be inconvenient to use this way and it is less flexible. Potentially, the way to perform it somehow, 
without any support of this, may be an argument against of using this connector.